src/_colors.scss

```diff
@@ -1,6 +1,6 @@
 $lightPurple: #b78add;
 $darkPurple: #751cc1;
-$lightGrey: #c3cfda;
-$darkGrey: #f7f9fa;
+$darkGrey: #9eadba;
+$lightGrey: #dfe6ee;
```

Comment:
I think I changed it again while adding the navbar because it was too dark in comparison with the navbar. I used a color picker on the design. But let's check it out together!